doc/modules/gaussian_process.rst

```diff
@@ -470,13 +470,13 @@ It is defined as:
 Kernel operators
 ----------------
 Kernel operators take one or two base kernels and combine them into a new
-kernel. The :class:`Sum` kernel takes two kernels :math:`k1` and :math:`k2`
-and combines them via :math:`k_{sum}(X, Y) = k1(X, Y) + k2(X, Y)`.
-The  :class:`Product` kernel takes two kernels :math:`k1` and :math:`k2`
-and combines them via :math:`k_{product}(X, Y) = k1(X, Y) * k2(X, Y)`.
+kernel. The :class:`Sum` kernel takes two kernels :math:`k_1` and :math:`k_2`
+and combines them via :math:`k_{sum}(X, Y) = k_1(X, Y) + k_2(X, Y)`.
+The  :class:`Product` kernel takes two kernels :math:`k_1` and :math:`k_2`
+and combines them via :math:`k_{product}(X, Y) = k_1(X, Y) \cdot k_2(X, Y)`.
 The :class:`Exponentiation` kernel takes one base kernel and a scalar parameter
-:math:`exponent` and combines them via
-:math:`k_{exp}(X, Y) = k(X, Y)^\text{exponent}`.
+:math:`p` and combines them via
+:math:`k_{exp}(X, Y) = k(X, Y)^p`.
```

Comment:
But this one kernel `k` can have input X and Y.
maybe the confusing part here is that `k_exp` doesn't take as input X and Y but a kernel k (that can have input X and Y) and outputs another kernel (that also can have input X and Y).

Reply:
Yes. correct. So we can leave it as is. It's just a bit confusing.

Reply:
Alternatively, it could be written in a more functional notation, `f_exp : Kernel space -> Kernel space`, `f_exp(k) = k^p=k(X,Y)^p` but I think the other might be less confusing for most people

Reply:
yeah, that's not our convention, doesn't fit.